guides/common/modules/proc_enabling-the-dhcp-module.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Enabling_the_DHCP_Module_{context}"]
```

Comment:
I think this should be covered by our other guides already, by relying on the installer.

Reply:
Removed as duplicate.